metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
The two concerns I have is that it sounds a little bit wiered (but I'm not the native one ;-)) and it somehow implies, there is also a response_time_millis

@tsg @monicasarbu Do you know if it was it ever an issue in packetbeat that responsetime is in milliseconds?